packages/amplify-category-api/src/provider-utils/awscloudformation/ecs-apigw-stack.ts

```diff
@@ -1,5 +1,7 @@
-import * as apigw2 from '@aws-cdk/aws-apigatewayv2';
-import * as cdk from '@aws-cdk/core';
+import * as apigw2 from 'aws-cdk-lib/aws-apigatewayv2';
+import * as apigw2alpha from '@aws-cdk/aws-apigatewayv2-alpha';
```

Comment:
I added this for constant definitions used below. We should be fine as this is internal usage.
The alternative is to copy constants into our codebase and maintain them.